sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Listeners/EventHubMetricsProvider.cs

```diff
@@ -78,9 +78,9 @@ public async Task<EventHubsTriggerMetrics> GetMetricsAsync()
             {
                 checkpoints = await Task.WhenAll(checkpointTasks).ConfigureAwait(false);
             }
-            catch
+            catch (Exception e)
             {
-                // GetCheckpointsAsync would log
+                _logger.LogWarning($"Encountered an exception while checking EventHub '{_client.EventHubName}'. Error: {e.Message}");
```

Comment:
```suggestion
                _logger.LogWarning($"Encountered an exception while checking Event Hub metrics for '{_client.EventHubName}'. Error: {e.Message}");
```

We check an Event Hub for many things.  The context won't be clear without some mention of metrics or scaling.

Reply:
Done -- added comment